wordclock/settings.h

```diff
@@ -46,8 +47,6 @@ typedef struct {
     uint32_t magic;
     EAnimationType animation;
     ETextEffect textEffect;
-    EHardwareType hardwareType;
-    EPixelType pixelType;
     uint32_t perfectImperfections;
```

Comment:
Goeie aktie, nuttig! Van mij mag je wel t/m `EOtaFwType otaFwType` verplaatsen. Die settings raak ik ook de hele tijd kwijt bij een update...

Reply:
Als ik dat doe dan ga ik een beetje voorbij aan mijn actie van het spiltten van de settings.
Het verliezen van de settings kunnen we ook oplossen door daar een stukje reserved te maken zodat nieuwe settings daar in komen.
Dan hoef je alleen de magic aan te passen als het type of de size van de huidige settings veranderd 
Ohh je verliest je settings steeds doordat de magic de laatste tijd veel veranderd is